ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,78 +16,247 @@
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SetProperty;
+import io.confluent.ksql.parser.tree.Statement;
+import io.confluent.ksql.parser.tree.UnsetProperty;
+import io.confluent.ksql.serde.DataSource.DataSourceType;
 import io.confluent.ksql.util.KsqlConfig;
+import io.confluent.ksql.util.KsqlException;
+import io.confluent.ksql.util.Pair;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 import org.apache.kafka.test.TestUtils;
 import org.easymock.EasyMock;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
 import java.util.Collections;
 
 import io.confluent.ksql.KsqlEngine;
 import io.confluent.ksql.function.UdfLoader;
 import io.confluent.ksql.util.PersistentQueryMetadata;
 import io.confluent.ksql.util.QueryMetadata;
 
+import static org.easymock.EasyMock.anyObject;
 import static org.easymock.EasyMock.anyString;
 import static org.easymock.EasyMock.eq;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
 public class StandaloneExecutorTest {
 
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final KsqlEngine engine = EasyMock.niceMock(KsqlEngine.class);
   private final UdfLoader udfLoader = EasyMock.niceMock(UdfLoader.class);
-  private final String query = "select * from bar;";
-  private StandaloneExecutor executor;
+  private String queriesFile;
+  private StandaloneExecutor standaloneExecutor;
+  private MetaStore metaStore;
 
   @Before
   public void before() throws IOException {
-    final String queriesFile = TestUtils.tempFile().getPath();
-    executor = new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
-    try(final FileOutputStream out = new FileOutputStream(queriesFile)) {
-      out.write(query.getBytes(StandardCharsets.UTF_8));
-    }
+    queriesFile = TestUtils.tempFile().getPath();
+    standaloneExecutor =
+        new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
+    metaStore = EasyMock.niceMock(MetaStore.class);
   }
 
+
   @Test
-  public void shouldCreateQueries() {
-    EasyMock.expect(engine.createQueries(query + "\n", ksqlConfig))
+  public void shouldRunCsStatement() throws IOException {
+
+    final List<Pair<String, Statement>> statementPairs = new ArrayList<>();
+
+    final CreateStream cs = new CreateStream(EasyMock.niceMock(QualifiedName.class), Collections.emptyList(), false, Collections.emptyMap());
+    statementPairs.add(new Pair<>("CS", cs));
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(statementPairs);
+    EasyMock.expect(engine.getMetaStore()).andReturn(metaStore);
+    final Map<String, Object> props = new HashMap<>();
```

Comment:
Duplicated line in each test. Just create static final EMPTY_PROPS = ImmutableMap.of();